CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ if(NOT PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
   file(GENERATE OUTPUT .gitignore CONTENT "*")
 endif()
 
+# Create compile_commands.json file
+set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
```

Comment:
ok, justification accepted.

please change to `option(CMAKE_EXPORT_COMPILE_COMMANDS "somedescription" ON)` to allow disabling it for the ones that want to.

Reply:
@akallabeth this doesn't work as it is already an internal CMake option :-(

Reply:
replaced by https://github.com/FreeRDP/FreeRDP/pull/11252